lib/peep.rb

```diff
@@ -0,0 +1,16 @@
+require 'peep'
+
+class Peep
+
+  include DataMapper::Resource
+
+  property :id, Serial
+  property :body, Text
+  property :created_at, DateTime
+
+  validates_presence_of :body
+  validates_length_of :body, :maximum => 200
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)